ESP-IDF v4.4 support

```
make[1]: *** No rule to make target 'true'.  Stop.
make: *** [/home/l33tname/esp/esp-idf/make/project.mk:649: component-mruby_component-build] Error 2
``` 

Does not work with the latest ESP-IDF v4.4.
I think or im holding it wrong. But i don't really understand the error here with true which should be a target.

Tested with v4.2.3 which worked fine.

--- a/components/mruby_component/component.mk
+++ b/components/mruby_component/component.mk
@@ -1,15 +1,16 @@
-
-COMPONENT_OWNBUILDTARGET := true
-COMPONENT_OWNCLEANTARGET := true
+COMPONENT_OWNBUILDTARGET := build
+COMPONENT_OWNCLEANTARGET := clean
 
 COMPONENT_ADD_INCLUDEDIRS := mruby/include/ mruby/build/esp32/include/
 
 CFLAGS += -Wno-char-subscripts -Wno-pointer-sign
 
+.PHONY: build
 build: 
 	cd $(COMPONENT_PATH)/mruby && MRUBY_CONFIG=../esp32_build_config.rb $(MAKE)
 	cp $(COMPONENT_PATH)/mruby/build/esp32/lib/libmruby.a $(COMPONENT_LIBRARY)
 
+.PHONY: clean
 clean: 
 	cd $(COMPONENT_PATH)/mruby && MRUBY_CONFIG=../esp32_build_config.rb $(MAKE) clean
 

